explorer/src/queries/search.ts

```diff
@@ -23,20 +43,13 @@ const searchBuilder = (searchQuery?: string): SelectQueryBuilder<JobRun> => {
   let query = getRepository(JobRun).createQueryBuilder('job_run')
 
   if (searchQuery != null) {
-    const searchTokens = searchQuery.split(/\s+/)
-    const normalizedSearchTokens = searchTokens.map(normalizeSearchToken)
+    let searchTokens = searchQuery.split(/\s+/)
+    searchTokens = searchTokens.concat(searchTokens.map(normalizeSearchToken))
     query = query
-      .where('job_run.runId IN(:...searchTokens)', { searchTokens })
-      .orWhere('job_run.jobId IN(:...searchTokens)', { searchTokens })
-      .orWhere('job_run.requester IN(:...normalizedSearchTokens)', {
-        normalizedSearchTokens,
-      })
-      .orWhere('job_run.requestId IN(:...normalizedSearchTokens)', {
-        normalizedSearchTokens,
-      })
-      .orWhere('job_run.txHash IN(:...normalizedSearchTokens)', {
-        normalizedSearchTokens,
-      })
+      .addSelect('COUNT(1) OVER() AS "totalRecords"')
```

Comment:
Would `COUNT(*)` be better? (I haven't seen (1) before, what does it do?)

Reply:
Ah, they are equivalent but it's good to stay consistent across the project so I'll update this.